include/vctr/Containers/VctrBase.h

```diff
@@ -259,7 +278,7 @@ class VctrBase : private Config,
 
```

Comment:
This description (above) does not seem to fit the function?

Reply:
I think the description might be a bit ambiguous if you only read the last line of the comment displayed here. The function that is meant here is the function object `fn` passed as an argument, which is expected to return something rather than that `forEach` member function. Does this become clearer to you if you read the entire comment or would you recommend rewording it?

Reply:
No, I actually mean the first line of the comment (hidden here and not affected by your PR): "Calls a function on each element and passes the element index as second function argument." – But actually it does not pass the index but the `Args` provided in the parameter pack when calling `forEach()`.

Reply:
Ah now I see. Yep, a copy & paste error. Will fix that :)